README.md

```diff
@@ -17,6 +21,13 @@ born to bring that same vision to the other platforms supported by Ebitengine.
 - **Dynamic Linking**: Load symbols at runtime and use it as a plugin system.
 - **Foreign Function Interface**: Call into other languages that are compiled into shared objects.
 
+## Supported Platforms
+
+- **Darwin**: amd64, arm64
```

Comment:
Maybe macOS is better? (If we want to use GOOS names, let's unify them with lower cases)